flake.lock

```diff
@@ -98,6 +98,21 @@
         "type": "github"
       }
     },
+    "flake-utils_2": {
+      "locked": {
+        "lastModified": 1642700792,
+        "narHash": "sha256-XqHrk7hFb+zBvRg6Ghl+AZDq03ov6OshJLiSWOoX5es=",
```

Comment:
Changes in this file are the same as on my PR #3143, which might cause conflicts if git is not smart enough.

Reply:
Yeah sorry I think I forgot to commit the lock file last time I modified the flake.nix file. I will merge master again if you get your changes in first